src/components/touchableOpacity/index.tsx

```diff
@@ -79,14 +79,22 @@ class TouchableOpacity extends PureComponent<Props, {active: boolean}> {
 
   onPressIn = (...args: any) => {
     this.setState({active: true});
-    //@ts-expect-error
-    this.props.onPressIn?.(...args);
+    if (this.props?.customValue) {
+      this.props.onPressIn?.(this.props.customValue);
+    } else {
+      //@ts-expect-error
+      this.props.onPressIn?.(...args);
+    }
   };
 
   onPressOut = (...args: any) => {
     this.setState({active: false});
-    //@ts-expect-error
-    this.props.onPressOut?.(...args);
+    if (this.props?.customValue) {
```

Comment:
same here (:

Reply:
Last week we agreed to merge without changing.

Reply:
We still need to pass the customValue inside on an object. You can pass all props, see what we do in onPress